404.html

```diff
@@ -1,16 +1,16 @@
 ---
 permalink: /404.html
 ---
- <!DOCTYPE html>
+<!DOCTYPE html>
 <html>
   <head>
```

Comment:
Remove the head tags (and the contents of it), and remove the html and body tags as these are handled by the layout